### ISSUE TITLE
control status of circulation pump

### ISSUE THREAD

**Issue by thomasklein1975** [gazoodle/geckolib#39]
Hi,

is it atm possible to control the status (switch on/off) of the circulation pump?

**Info about my System/Config:**

`Heater: Temperature 33.5°C, SetPoint 32.0°C, Real SetPoint 32.0°C, Operation Cooling
Waterfall: OFF
Pump 1: OFF
Pump 2: OFF
Lights: OFF
WaterCare: Waiting...
Smart Winter Mode:Risk: NO
Circulating Pump: OFF
Ozone: OFF
Smart Winter Mode:Active: False
Filter Status:Clean: False
Filter Status:Purge: False
Economy Mode: False
Error Sensor: OverTemp
Welcome to the Gecko shell. Type help or ? to list commands.

Wellness$ help

Documented commands (type help <topic>):
========================================
LI         about      exit     list      monitor  setpoint  watercare
P1         accessors  get      logfile   peek     snapshot
P2         discover   help     loglevel  refresh  state
Waterfall  eco        license  manage    set      version

Wellness$ list
1. Wellness
Wellness$ version
geckolib version 0.4.8
SpaPackStruct.xml revision 36.01
intouch version EN 88 v15.0
intouch version CO 89 v11.0
Spa pack inYT 380 v4.0
Low level configuration # 6
Config version 65
Log version 66
Pack type 10
`

**Comment by gazoodle** [comment]
I don’t think this will be possible as there is no user demand control for it as it is turned on/off automatically by the spa for heating and filtering.  I think we’re only ever going to be able to get a sensor for it. Some spas have it as part of the waterfall feature, in which case the pump can be turned on/off if the pack doesn’t want it on. 

I’ll be putting this knowledge into the next release of the library.
